Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:2.7-slim
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update -qq && \
+  apt-get install -y --no-install-recommends \
+  build-essential && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+  mkdir /app
+
+WORKDIR /app
+
+# Copy as early as possible so we can cache ...
+ADD requirements.txt .
```

Comment:
i think `COPY` is the better choice here (as we don't need fancy `ADD` features)